FreeRTOS/FreeRTOS
# Add unit tests for GetStaticBuffer functions

Description
-----------


Add unit tests for the various `...GetStaticBuffer()` functions added in https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/641. The `...GetStaticBuffer()` functions are test for the following behaviors:
- For static objects,  `...GetStaticBuffer()` returns `pdTRUE` and static object's buffers
- For dynamic objects, `...GetStaticBuffer()` returns `pdFALSE` and leaves the return arguments unchanged.

Test Steps
-----------


```
cd FreeRTOS/Test/CMock/
make
```

Checklist:
----------



Related Issue
-----------


https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/641

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
